vizro-ai/pyproject.toml

```diff
@@ -48,4 +49,8 @@ parallel = true
 source_pkgs = ["vizro_ai"]
 
 [tool.pytest.ini_options]
-filterwarnings = "error"
+filterwarnings = [
+  "error",
+  # Ignore this warning here as it comes from pandas until pandas is made compatible with Python 3.12
+  'ignore:.*utcfromtimestamp.*:DeprecationWarning'
```

Comment:
Which tests raise this warning or is it all over the place?

Reply:
Like the imports of the tests don't even work:

```
ImportError while loading conftest '/Users/huong_li_nguyen/Box Sync/GitRepos/vizro/vizro-core/tests/conftest.py'.
tests/conftest.py:3: in <module>
    from vizro import Vizro
src/vizro/__init__.py:4: in <module>
    from ._vizro import Vizro
src/vizro/_vizro.py:9: in <module>
    from vizro.managers import data_manager, model_manager
src/vizro/managers/__init__.py:2: in <module>
    from ._data_manager import data_manager
src/vizro/managers/_data_manager.py:4: in <module>
    import pandas as pd
```